src/main/webapp/teacher/controlGameInstance/controlGameInstance.js

```diff
@@ -74,15 +98,15 @@ function queryActiveGameInstanceDocument(gameInstanceId) {
 }
 
 // Build the UI elements with all the info of the GameInstance
-function buildActiveGameInstanceUI(gameInstance, gameInstanceId) {
+function buildActiveGameInstanceUI(gameInstanceId, gameInstance, game) {
```

Comment:
This would benefit from using named parameters so you don't have to worry about argument ordering. The common way people do named parameters in JS is this.

```js
function buildActiveGameInstanceUI({ gameInstanceId, gameInstance, game } = {}) {...}

buildActiveGameInstanceUI({ gameInstanceId: 123, gameInstance: gameInstance, game: game });

// Which can be shortened to this
buildActiveGameInstanceUI({ gameInstanceId: 123, gameInstance, game });
```

Reply:
So nice!! So in the last example I could call `buildActiveGameInstanceUI` with just `gameInstanceId, gameInstance, game` in any order (as long as they're named as the parameters of the function)?

Reply:
Like this: `buildActiveGameInstanceUI({ game, gameInstanceId, gameInstanceId });`